packages/neuron-wallet/src/types/cell-types.ts

```diff
@@ -13,6 +13,11 @@ export interface BlockHeader {
   number: string
 }
 
+export enum TransactionStatus {
```

Comment:
Consider using `pending` (or `sending` or `receiving`), `success` and `failed`.

Reply:
ok

Reply:
updated